test.js

```diff
@@ -21,6 +21,15 @@ test('main', t => {
 	t.is(cliTruncate('unicorns', 6, {position: 'middle'}), 'uni…ns');
 });
 
+test('preferTruncationOnSpace option', t => {
+	t.is(cliTruncate('unicorns rainbow dragons', 20, {position: 'start', preferTruncationOnSpace: true}), '…rainbow dragons');
+	t.is(cliTruncate('unicorns rainbow dragons', 20, {position: 'end', preferTruncationOnSpace: true}), 'unicorns rainbow…');
+	t.is(cliTruncate('unicorns rainbow dragons', 6, {position: 'start', preferTruncationOnSpace: true}), '…agons');
+	t.is(cliTruncate('unicorns rainbow dragons', 6, {position: 'end', preferTruncationOnSpace: true}), 'unico…');
+	t.is(cliTruncate('unicorns rainbow dragons', 6, {position: 'middle', preferTruncationOnSpace: true}), 'uni…ns');
+	t.is(cliTruncate('unicorns rainbow dragons', 20, {position: 'middle', preferTruncationOnSpace: true}), 'unicorns…dragons');
```

Comment:
Try with some different kinds of strings. Look at my feedback in https://github.com/sindresorhus/cli-truncate/pull/10.